src/MIDebugEngine/Engine.Impl/Variables.cs

```diff
@@ -90,6 +90,19 @@ internal sealed class VariableInformation : IVariableInformation
         private string DisplayHint { get; set; }
         public bool IsPreformatted { get; set; }
 
+        public string Address()
+        {
+            // ask GDB to evaluate "&expression"
+            string command = "&("+FullName()+")";
+            return EvalDependentExpression(command);
```

Comment:
>return [](start = 12, length = 6)

We should probably verify the result we gets back looks like an address.

Reply:
You mean have a conditional that checks for "0x" at the beginning? If so, ohh. I think I misunderstood it before and physically checked for it.

Reply:
Yeah, I would think we would want to verify the result is either:
* `^0x[0-9a-fA-F]+$`
-or-
* `^(0x[0-9a-fA-F]+)\b` and return the captured string